confirmation/models.py

```diff
@@ -232,7 +232,7 @@ class Confirmation(models.Model):
     UNSUBSCRIBE = 4
     SERVER_REGISTRATION = 5
     MULTIUSE_INVITE = 6
-    REALM_CREATION = 7
+    REALM_ACTIVATION = 7
```

Comment:
Perhaps we want `REALM_REGISTRATION` rather than `REALM_ACTIVATION`, to parallel `USER_REGISTRATION`? Or maybe `REALM_CREATION` is fine for that one. 

I think the former `RealmCreationKey` might be better as something like `CAN_CREATE_REALM`? It's a permission to, once, create a realm, in a server that restricts that capability.

Reply:
Yeah I think we can rename that and just put a small mention in upgrade notes. 

> The obscure  `REALM_CREATION_LINK_VALIDITY_DAYS` setting was renamed to `CAN_CREATE_REALM_LINK_VALIDITY_DAYS`.  

There's no reason to change that setting, so probably it will not impact anyone.

I think the one current called `REALM_CREATION` would be best named `NEW_REALM_USER_REGISTRATION` to parallel `USER_REGISTRATION` -- since it's confirming the same thing.